api/src/services/healthService.js

```diff
@@ -94,6 +97,44 @@ function splitParams(params) {
   return params && params.split(SEPARATOR)
 }
 
+async function liveness(req, res) {
+  res.status(200).json({
+    msg: 'Liveness probe is ok',
+  });
+}
+
+async function readiness(req, res) {
+  let errors = [];
+  
+    let client;
+    try {
+      client = await getClient();      
+    } catch (error) {
+      console.error(error);
+      errors.push(`Unable to connect to database:${err.message}`);
+    } finally {
+      client.end();
```

Comment:
`client` can be `undefined`. Please, add a check.

Reply:
If an error happened  during client generation then it fill be undefined